Sources/Apodini/Semantic Model Builder/REST/RESTEndpointHandler.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 @_implementationOnly import Vapor
 import protocol FluentKit.Database
 
-extension Vapor.Request: ExporterRequest, WithEventLoop {}
+extension Vapor.Request: ExporterRequest, WithEventLoop, Reducible {}
```

Comment:
If `Reducible` is a required protocol, can we make so that the Exporter doesn't need to specify the conformance explicitly? I think that would do a good job hiding this complexity to Exporter which don't use aggregated Requests.

Reply:
You are right, this was a bit ugly. I fixed it. (When I first tried, I had some problems with `Self` as a protocol requirement and then forgot about this convenience extension when I resolved that issue.)